clients/da-vinci-client/src/main/java/com/linkedin/davinci/client/AvroGenericDaVinciClient.java

```diff
@@ -700,6 +701,10 @@ private VeniceConfigLoader buildVeniceConfig() {
       kafkaBootstrapServers = backendConfig.getString(KAFKA_BOOTSTRAP_SERVERS);
     }
 
+    String recordTransformerOutputValueSchema = daVinciConfig.isRecordTransformerEnabled()
+        ? Objects.toString(recordTransformerConfig.getOutputValueSchema(), "null")
+        : "null";
```

Comment:
The ternary makes sense if you wanted to set with proper null, but as is this is redundant.